### PR TITLE
Update ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by gibfender**
missing `

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -16,4 +16,4 @@
 - Dedicated / Self-Hosted Multiplayer
 
 **RPT log file:**
-- Add a link ([gist](https://gist.github.com) or [pastebin](http://pastebin.com)) to the client and/or server RPT file. An instruction to find your RPT files can be found [here](https://community.bistudio.com/wiki/Crash_Files#Arma_3).
\ No newline at end of file
+- Add a link ([gist](https://gist.github.com) or [pastebin](http://pastebin.com)) to the client and/or server RPT file. An instruction to find your RPT files can be found [here](https://community.bistudio.com/wiki/Crash_Files#Arma_3).
```
Comment (dedmen) [inline .github/ISSUE_TEMPLATE.md]: What changed here?

Comment (gibfender) [inline .github/ISSUE_TEMPLATE.md]: I didn't touch that section, not sure why it's flagged a change

Comment (dedmen) [inline .github/ISSUE_TEMPLATE.md]: Can you try to revert it? probably changed line ending

Comment (gibfender) [inline .github/ISSUE_TEMPLATE.md]: Apologies, i'm not sure how to do that! If it helps nothing appears to change if you look at the rich diff

Comment (dedmen) [inline .github/ISSUE_TEMPLATE.md]: Github inside webbrowser seems to not be able to fix it... I'll do that tomorrow then...

**Comment by dedmen** [comment]
What a hassle...

**Pull request merged**